src/runtime/pkg/oci/utils.go

```diff
@@ -362,6 +363,91 @@ func SandboxID(spec specs.Spec) (string, error) {
 	return "", fmt.Errorf("Could not find sandbox ID")
 }
 
+func addContainerAnnotations(ocispec specs.Spec, _config *vc.ContainerConfig) error {
```

Comment:
This is add container device from annotation right? Not add container annotation

Reply:
Yes, this function is for updating the container's spec or config, now it only does such work for devices, but for future usage, I chose this name, according to the same naming style for updating sandbox config.